sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 #### Bugs Fixed
 
 #### Other Changes
+* Remove requires io.netty.transport.epoll from module-info.
+* Converted from microseconds to milliseconds in CosmosDiagnostics (i.e. replaced "durationInMicroSec" with to "durationInMilliSecs"; and changed type from long to double) - See See [PR 29643](https://github.com/Azure/azure-sdk-for-java/pull/29643)
```

Comment:
I do not think we need to mention the type change, so thinking the line can be shortened into:

Converted from `durationInMicroSec` to `durationInMilliSecs` in `CosmosDiagnostics` - See  [PR 29643](https://github.com/Azure/azure-sdk-for-java/pull/29643)

We generally need to highlight the keywords, for example from CosmosDiagnostics to `CosmosDiagnostics`

Reply:
Shortened as requested.